src/shipit_code_coverage_backend/shipit_code_coverage_backend/coverage.py

```diff
@@ -45,6 +47,23 @@
                 }
 
 
+async def get_push_id(changeset):
+    '''
+    Get push ID for a given changetet, using ActiveData when available
+    '''
+    try:
```

Comment:
Instead of try catch, can we check if `get_push` is available and use it if it is or hgweb if it isn't?